model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/velocity/velocity_advection.py

```diff
@@ -557,6 +784,17 @@ def run_corrector_step(
         # This behaviour needs to change for multiple blocks
         self.levelmask = self.levmask
 
+        """
+        ddt_vn_apc_pc[ntnd] (0:nlev-1):
+            Compute the advection of normal wind at full levels (edge center).
+            ddt_vn_apc_pc[ntnd] = dKEH/dn + vt * (vorticity + coriolis) + wdvn/dz,
+            where vn is normal wind, vt is tangential wind, KEH is horizontal kinetic energy.
+            z_kin_hor_e is KEH at full levels (edge center).
+            zeta is vorticity at full levels (vertex). Its value at edge center is simply taken as average of vorticity at neighboring vertices.
+            w, which is vertical wind with contravariant correction, at edge center is obtained by linearly interpolating z_w_con_c_full at neighboring cells.
+            dvn/dz, the vertical derivative of normal wind, is computed by first order discretization from vn_ie.
+            TODO (Chia Rui): understand the coefficient coeff_gradekin
```

Comment:
```suggestion
            ddt_vn_apc_pc[ntnd] = dKEH/dn + vt * (vorticity + coriolis) + wdvn/dz,
            where vn is normal wind, vt is tangential wind, KEH is horizontal kinetic energy.
            z_kin_hor_e = KEH at full levels (edge center).
            zeta = vorticity at full levels (vertex). Its value at edge center is an average of vorticity at neighboring vertices.
            w = vertical wind with contravariant correction. Its value at edge center is obtained by linearly interpolating z_w_con_c_full at neighboring cells.
            dvn/dz = vertical derivative of normal wind, computed by first order discretization from vn_ie.
            TODO (Chia Rui): understand the coefficient coeff_gradekin
```

Reply:
This is more or less out convention in other docstrings. Can you adapt documentation in other places in this PR plase?